ipa-core/src/net/mod.rs

```diff
@@ -24,6 +25,10 @@ pub use transport::{HttpShardTransport, HttpTransport};
 
 pub const APPLICATION_JSON: &str = "application/json";
 pub const APPLICATION_OCTET_STREAM: &str = "application/octet-stream";
+pub static HTTP_CLIENT_ID_HEADER: HeaderName =
```

Comment:
this name is really confusing, we should clearly indicate the difference between MPC-to-MPC and shard-to-shard authentication headers.

Reply:
I was following the names of the respective structs: `ClientIdentity` and `ShardIndex`.  Do you think we really need to change this?

Reply:
This PR seems to  change `ClientIdentity` meaning to be able to represent both shard and MPC origins, so I don't think it is tied up to MPC connections only.

Reply:
sorry, I meant to say `HelperIdentity` instead of `ClientIdentity`. Should I rename to `HTTP_HELPER_ID_HEADER` instead?